### ISSUE TITLE
Screen reader does not work

### ISSUE THREAD

**Issue by jepst** [linuxmint/Cinnamon#3187]
Turning on the screen reader (either via the keyboard shortcut or via the GUI in the Accessibility control panel) has no effect. The screen reader does not start.

Orca is installed. It can be activated manually (by running orca from the command line) but not via Cinnamon.

Cinnamon 2.2.10+qiana on Mint RC 17.

**Comment by coffeeking** [comment]
I can confirm this happens in cinnamon 2.6. Starting orca manually does work, but the universal shortcut alt+super+s does not. Mate got around this by checking if the key org.gnome.applications screen-reader-enabled was set to true. If not, it toggles it. This should be easier once orca switches to use gsettings for it's configs, but this has not happened yet.

**Comment by Vahan86** [comment]
@jepst, @coffeeking, is this still an issue for you with Cinnamon 3.0.x?

**Comment by Vahan86** [comment]
@JosephMcc, can we close the issue?
